src/docs/cookbook/gestures/dismissible.md

```diff
@@ -70,27 +73,29 @@ such as removing the item from a web service or database.
 
 Update the `itemBuilder()` function to return a `Dismissible` widget:
 
-<!-- skip -->
+<?code-excerpt "lib/step2.dart (Dismissible)"?>
 ```dart
-Dismissible(
-  // Each Dismissible must contain a Key. Keys allow Flutter to
-  // uniquely identify widgets.
-  key: Key(item),
-  // Provide a function that tells the app
-  // what to do after an item has been swiped away.
-  onDismissed: (direction) {
-    // Remove the item from the data source.
-    setState(() {
-      items.removeAt(index);
-    });
-
-    // Show a snackbar. This snackbar could also contain "Undo" actions.
-    ScaffoldMessenger
-        .of(context)
-        .showSnackBar(SnackBar(content: Text("$item dismissed")));
-  },
-  child: ListTile(title: Text('$item')),
-);
+itemBuilder: (context, index) {
+  final item = items[index];
+  return Dismissible(
```

Comment:
I've included the part that shows the `itemBuilder` so it is clear that the `Dismissible` is used inside.